pylmod/gradebook.py

```diff
@@ -1130,3 +1139,17 @@ def get_staff(self, gradebook_id, simple=False):
                 })
             return simple_list
         return staff_data['data']
+
+    def get_grade_multipliers(self):
+        """
+        Use cached data from self.get_assignments() to provide the
+        grade multipliers (maxPointsTotal value).
+        """
+        if self._assignment_cache is None:
+            # fill cache
+            self.get_assignments()
+
+        return {
+            x["assignmentId"]: x["maxPointsTotal"]
```

Comment:
We ought to confirm that maxPointsTotal always contains a valid number.  It would be catastrophic if it can be zero.  :-(